internal/resources/aws/dynamodb_table.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/infracost/infracost/internal/usage/aws"
 )
 
+type PointInTimeRecovery struct {
+	Enabled bool `infracost_usage:"enabled"`
+}
+
```

Comment:
I'm not sure why you have usage keys here, I don't think this works. The `infracost_usage` relates to the specific key in the usage file under `aws_dynamodb_table`. 

1. `enabled` is not defined
2. the nesting is not handled.

but I don't believe usage is required here as this attribute is inferred from the code

Reply:
yup, removed - misunderstanding on my part about what that annotation was for when I wrote it